v2/src/modules/address/ModuleAddressTokens.vue

```diff
@@ -216,9 +227,10 @@ const { renderState } = useAppTableRowRender(tokensLength.value)
 
 const tokens = computed(() => {
     if (!loadingTokens.value && hasTokens.value && tokenSort.value) {
-        return renderState.isActive
-            ? tokenSort.value?.getSortedTokens(state.sortKey).slice(0, renderState.maxItems)
-            : tokenSort.value?.getSortedTokens(state.sortKey)
+        const tokenSorted = tokenSort.value?.getSortedTokens(state.sortKey)
+        const [startsWith, notStartsWith] = searchHelper(tokenSorted, ['name', 'symbol', 'contract'], state.searchParams)
```

Comment:
add  an if so you don't use that function every time 
`if (state.searchParams) !== ''`

 once  spread operator is added
`  const tokens = searchHelper(tokenSorted, ['name', 'symbol', 'contract'], state.searchParams)`